docs/whats_new.rst

```diff
@@ -262,5 +297,6 @@ Previous Releases of FLARE
 .. toctree::
    :maxdepth: 1
 
+   release_notes/flare_230
    release_notes/flare_220
    release_notes/flare_210
```

Comment:
I am wondering do we want to include the PR/commits as shown in github: https://github.com/NVIDIA/NVFlare/releases/tag/2.3.5 

And do we want to include minor version release notes?

Reply:
Sure I can add a link to the github releases page. Minor version release notes could be useful if there is a significant rc